app/components/CollapsibleDisplayContent/index.tsx

```diff
@@ -51,15 +51,17 @@ function CollapsibleDisplayContent({
       }}
     >
       <div key={content} id={id} style={style} className={className} ref={ref}>
-        <Editor
-          onChange={() => {}}
-          onBlur={() => {}}
-          onFocus={() => {}}
-          value={content}
-          placeholder={placeholder}
-          disabled
-          domParser={domParser}
-        />
+        <i>
```

Comment:
I'm a bit negative to "hardcoding" this to always be italic

Reply:
Why not just set `font-style: italic` on the `.noEventsMessage` class?

Reply:
Yeah, but on second thought I don't really think this text needs to be italic anyways